packages/nimble-components/src/wafer-map/tests/data-manager.spec.ts

```diff
@@ -63,36 +65,36 @@ describe('Wafermap Data manager', () => {
 
     it('computes the correct containerDimensions', () => {
         expect(dataManagerModule.containerDimensions).toEqual({
-            width: 60,
-            height: 60
+            width: 92,
+            height: 92
         });
     });
 
     it('computes the correct radius', () => {
```

Comment:
So, 92 is right because it's the provided height or width minus the two margins of the container.

These tests are the unit tests for the computations module and they should test the logic behind it, so that when a person not knowledgeable about it wants to make changes, they will be there to make sure the behaviour remains unchanged.

I think this was a big change because we modified a big part of how this module made calculations. In a use case where we would want the behaviour unchanged these tests should be our guarantee that it remains so.

I can make changes so we have computed values but I think that will be counterproductive because if the logic changes again then the computing logic for the tests will have to change, and that will not be as straightforward as just a few values. I hoped these were very clear usages as to where the values are coming from, if it helps, I could also add comments about what they represent

Reply:
Your point about the tests providing more value when the behavior isn't changing is a good one. I am ok to leave the tests as they are and keep an eye on whether they provide value for those sorts of changes.

Context about where values come from is always good. If you do want to improve that, I think the test description is the best place rather than separate comments. But I'll let you judge whether that's necessary; I honestly haven't taken a close look at that level of implementation detail.